open-machine-learning-jupyter-book/assignments/prerequisites/python-programming-basics.ipynb

```diff
@@ -1,12 +1,27 @@
 {
```

Comment:
<p>The class name should be like <code>TestStringUpper()</code></p><p>And you should add some more tests for the unhappy cases in the next PR. Please apply this to all your tests</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Please address this comment.</p>